app/assets/javascripts/charts.js

```diff
@@ -395,7 +395,7 @@ $(function() {
   $(document).on('click', '#legendContainer .legendColorBox, .legendContainer .legendLabel', function() { ext_legend_selected(this);});
 });
 
-$(window).on('resize', refreshCharts);
+// $(window).on('resize', refreshCharts);
```

Comment:
Disabled for development purposes. This and all files using the charts.js file need to be refactered because this line adds an event handler to the global window object and the charts.js file is loaded be the asset pipeline for every page that is loaded and the event handler runs on every page..